mmdeploy/codebase/mmpose/models/heads/yolox_pose_head.py

```diff
@@ -0,0 +1,219 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+from typing import List, Optional, Tuple
+
+import torch
+from mmengine.config import ConfigDict
+from mmengine.structures import InstanceData
+from torch import Tensor
+
+from mmdeploy.codebase.mmdet import get_post_processing_params
+from mmdeploy.core import FUNCTION_REWRITER
+
+
+def yolox_pose_head_nms(boxes: Tensor,
```

Comment:
could add a ut for this rewriting

Reply:
may update the docs https://github.com/open-mmlab/mmdeploy/blob/main/docs/en/04-supported-codebases/mmpose.md#supported-models